ci_framework/roles/ci_make/molecule/default/prepare.yml

```diff
@@ -0,0 +1,31 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Prepare
+  hosts: all
+  tasks:
+    - name: Create lame directory
+      ansible.builtin.file:
+        path: /tmp/foo
+        state: directory
+    - name: Push dummy Makefile
+      ansible.builtin.file:
+        target: /tmp/foo/Makefile
+        content: |
+          FOO_BAR ?= "toto"
+          help:
+            echo "This is the help thing showing ${FOO_BAR}"
```

Comment:
- name: push dummy makefile
  ansible.builtin.copy:
      dest:  /tmp/foo/Makefile
       content: |
          FOO_BAR ?= "toto"
          help:
            echo "This is the help thing showing ${FOO_BAR}"